modules/nf-scil/preproc/n4/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "preproc_n4"
+description: Bias field correction using N4
+keywords:
+  - "ANTs"
```

Comment:
Move in the tools section if you want to describe ANTs and scilpy. Else here just put some keywords.